base/docs/helpdb.jl

```diff
@@ -7463,9 +7463,10 @@ true
 all(p, itr)
 
 doc"""
-    bind(socket::Union{UDPSocket, TCPSocket}, host::IPv4, port::Integer)
+    bind(socket::Union{UDPSocket, TCPSocket}, host::IPAddr, port::Integer; ipv6only=false)
```

Comment:
Is the IPv6 fix logically connected to creating getsockname? If not, maybe it should be a separate PR?